controllers/datadogagent/controller.go

```diff
@@ -139,11 +142,15 @@ func (r *Reconciler) reconcileInstance(ctx context.Context, logger logr.Logger,
 	// -----------------------
 	// Manage dependencies
 	// -----------------------
-	depsStore := dependencies.NewStore(&dependencies.StoreOptions{SupportCilium: r.options.SupportCilium})
-	rbacManager := feature.NewResourcesManagers(depsStore)
+	storeOptions := &dependencies.StoreOptions{
+		SupportCilium: r.options.SupportCilium,
+		Logger:        logger,
+	}
+	depsStore := dependencies.NewStore(storeOptions)
+	resourcesManager := feature.NewResourcesManagers(depsStore)
```

Comment:
nit for a future PR, i think it'd be more appropriate to call the struct `ResourceManagers` (drop the `s` in `Resources`)